lib/splash.js

```diff
@@ -3,6 +3,8 @@ var _ = require("lodash");
 var level = 0;
 var _logger;
 
+var KEYS_TO_HIDE = ["pass", "secret"];
```

Comment:
In `av-ui` I have several keys to hide with names not containing these 2 words. It could be nice to move this array to `node-tech-logger` configuration to be able to customize it.
What do you think?